library/ix-dev/community/netbootxyz/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+runAsContext:
+  - userName: netboot
+    groupName: netboot
+    gid: 999
+    uid: 999
```

Comment:
A quick look inside the container, does not seem to have that user at all.
Also considering that it spawns multiple processes I'm not sure it will be able to run rootless